var/spack/repos/builtin/packages/proj/package.py

```diff
@@ -128,6 +128,7 @@ def install_datum_grids(self):
 
 class CMakeBuilder(BaseBuilder, cmake.CMakeBuilder):
     def cmake_args(self):
+        shared_arg = "BUILD_SHARED_LIBS" if self.spec.satisfies("@7:") else "BUILD_LIBPROJ_SHARED"
```

Comment:
You don't actually use this variable...

Reply:
Thanks for catching this, the change where I use this ended up in another branch on rebase, so you've fixed two mistakes.